frontend/lib/identification/card_detail_view.dart

```diff
@@ -48,70 +43,91 @@ class CardDetailView extends StatelessWidget {
                     ? CrossAxisAlignment.start
                     : CrossAxisAlignment.stretch,
                 children: [
-                  IdCard(
-                    height: isLandscape ? 200 : null,
-                    child: SvgPicture.asset("assets/card.svg",
-                        semanticsLabel: 'Ehrenamtskarte',
-                        alignment: Alignment.center,
-                        fit: BoxFit.contain),
-                  ),
-                  Container(
-                    padding: EdgeInsets.symmetric(horizontal: 4),
-                    alignment: Alignment.centerRight,
-                    child: InkWell(
-                        child: Text(
-                          "Code neu einscannen",
-                          style:
-                              TextStyle(color: Theme.of(context).accentColor),
-                        ),
-                        onTap: onOpenQrScanner),
+                  Padding(
+                    padding: const EdgeInsets.symmetric(vertical: 8.0),
+                    child: IdCard(
+                        height: isLandscape ? 200 : null,
+                        child: EakCard(
+                            cardDetails: cardDetails,
+                            region: Region(region.prefix, region.name))),
```

Comment:
Crashes when there is an issue getting the region so that it is null.

`#NullSafetyNow` `#I<3NullSafety` `#MakeTheWorldNullSafeAgain`